core/src/main/java/greencity/controller/SuperAdminController.java

```diff
@@ -462,6 +462,23 @@ public ResponseEntity<CourierDto> deactivateCourier(@PathVariable Long id) {
         return ResponseEntity.status(HttpStatus.OK).body(superAdminService.deactivateCourier(id));
     }
 
+    /**
+     * Controller for activate courier's.
+     *
+     * @param id - courier id that will need to be deleted;
+     */
+    @ApiOperation(value = "Activate courier's by Id")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
+    })
+    @PatchMapping("/activateCourier/{id}")
+    public ResponseEntity<CourierDto> activateCourier(@PathVariable Long id) {
```

Comment:
Why do we have different responses for courier and employee activation
We retund CourierDto for courier activation and only the response code for employee activation.